src/test/java/org/broadinstitute/consent/http/cloudstore/GCSServiceTest.java

```diff
@@ -139,4 +140,30 @@ void testDeleteDocument() {
     assertTrue(deleted);
   }
 
+  @Test
+  void testReadJsonFileFromBucket() throws Exception {
+    String testJson = """
+        {
+          "institutionDomainMap": {
+            "Broad Institute": ["broadinstitute.org", "broad.mit.edu"],
+            "Harvard": ["harvard.edu", "hms.harvard.edu"]
+          }
+        }
+        """;
+
+    String blobIdName = "institution-domain-mapping.json";
+    Blob blob = mock(Blob.class);
```

Comment:
`mock()` infers its return type from context so a parameter is unnecessary
```suggestion
    Blob blob = mock();
```

Although in this case there is already a mock `blob` field so I don't think this is necessary.